iceberg/iceberg-handler/src/test/results/positive/col_stats.q.out

```diff
@@ -339,17 +339,16 @@ POSTHOOK: type: DESCTABLE
 POSTHOOK: Input: default@tbl_ice_puffin
 col_name            	a                   
 data_type           	int                 
-min                 	1                   
```

Comment:
@simhadri-g, seems like the `desc formatted tbl_ice_puffin a` gets status from metastore though `hive.iceberg.stats.source=iceberg`, cloud you please check?

Reply:
@dengzhhu653 , 
Yes, we recently merged a PR to store hive column stats in puffin files for iceberg tables. https://github.com/apache/hive/commit/a8a0ae782be87d1198006ea3cb508f14070231b7 

There are few more follow up tasks that i am currently working as a part of the epic.